tensorflow/compiler/tf2tensorrt/utils/trt_shape_optimization_profiles.h

```diff
@@ -62,14 +62,21 @@ struct OptimizationProfileConfig {
   // network - TensorRT network, used to enumerate all the input tensors
   // profile - on exit the profile information will be set for each input tensor
   Status SetDimensions(const nvinfer1::INetworkDefinition* network,
```

Comment:
Now that SetDimensions is doing more than what it used to do, shall we rename it to AccumulateInputsAndSetDimensions. Please also modify the document for the routine to reflect this change.

Reply:
This is not accumulating dimensions, only setting them, end ensuring that they are consistent with the network definition. I have adjusted the docstring to explain this.